docker-compose-v1.yml

```diff
@@ -39,7 +39,3 @@ web:
   volumes:
       # This directory must have cert files
     - ./volumes/web/cert:/cert:ro
-    - /etc/localtime:/etc/localtime:ro
```

Comment:
sorry for the typo. Auto correct... I saw you added it back in docker-compose.yml. This is removed from docker-compose-v1.yml. Thanks.